jobs/e2e-test-cypress.groovy

```diff
@@ -128,14 +122,76 @@ def runLamdaTest() {
    dir('webapp/services/static') {
       withEnv(["LT_USERNAME=${lt_username}",
                "LT_ACCESS_KEY=${lt_access_key}"]) {
-         exec(runLambdaTestArgs); 
+         exec(runLambdaTestArgs);
+      }
+   }
+}
+
+def analyzeResults() {
+   withTimeout('5m') {
+      if (currentBuild.result == 'ABORTED') {
+         // No need to report the results in the case of abort!  They will
+         // likely be more confusing than useful.
+         echo('We were aborted; no need to report results.');
+         return;
+      }
+
+      dir('webapp/services/static') {
+         withCredentials([
+            string(credentialsId: "SLACK_BOT_TOKEN", variable: "SLACK_TOKEN")
+         ]) {
+            def deployerUsername = params.DEPLOYER_USERNAME;
+            
+            def notifyResultsArgs = [
```

Comment:
**note:** In the Python/Selenium script, we invoke a different script that was created specifically to process the custom Python results: https://github.com/Khan/jenkins-jobs/blob/777e1934c8204b466b01cdb51a832f5694181bc8/jobs/e2e-test.groovy#L560.

In our case, we created our own implementation so we can process the Cypress/LambdaTest results properly.